journal-of-universal-computer-science.csl

```diff
@@ -0,0 +1,529 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style class="in-text" version="1.0" and="text" delimiter-precedes-last="never" demote-non-dropping-particle="never" xmlns="http://purl.org/net/xbiblio/csl">
+<!-- This style was edited with the Visual CSL Editor (http://editor.citationstyles.org/visualEditor/) -->
+  <info>
+    <title>Journal of Universal Computer Science</title>
+    <title-short>J.UCS</title-short>
+    <id>http://www.zotero.org/styles/journal-of-universal-computer-science</id>
+    <link href="http://www.zotero.org/styles/journal-of-universal-computer-science" rel="self"/>
+    <link rel="template" href="http://www.zotero.org/styles/apa-single-spaced"/>
+    <link href="http://www.jucs.org/ujs/jucs/info/submissions/style_guide.html" rel="documentation"/>
+    <author>
+      <name>Georg Link</name>
+      <email>glink@unomaha.edu</email>
+    </author>
+    <category citation-format="author-date"/>
```

Comment:
Thank you. I had not found the list when I went looking for it and `computer_science` does not work.